tests/core/arch-arm64/stalker-arm64.c

```diff
@@ -1240,6 +1242,34 @@ TESTCASE (follow_thread)
 #endif
 }
 
+TESTCASE (pthread_create)
+{
+  int ret;
+  pthread_t thread;
+  int number = 0;
+  fixture->sink->mask = (GumEventType) (GUM_COMPILE);
+
+  gum_stalker_follow_me (fixture->stalker, fixture->transformer,
+      GUM_EVENT_SINK (fixture->sink));
+
+  ret = pthread_create (&thread, NULL, increment_integer, (gpointer) &number);
+  g_assert_cmpuint (ret, ==, 0);
```

Comment:
```suggestion
  g_assert_cmpint (ret, ==, 0);
```

As it's signed, not unsigned.